.github/CODEOWNERS

```diff
@@ -0,0 +1,3 @@
+*.rst @docs
```

Comment:
I think this should be `@crate/docs`, at least in other repos we use the `@crate/` prefix.